Octokit/Models/Request/ApiOptions.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class ApiOptions
```

Comment:
I'm somewhat concerned about this becoming a "bucket" to put all the stuff we don't really know how to put elsewhere, but ultimately this is what the user should pass through to the library to tweak the defaults.